support/mesh/mesh_protobuf/src/lib.rs

```diff
@@ -20,8 +20,15 @@
 #![warn(missing_docs)]
 // UNSAFETY: Serialization and deserialization of structs directly.
 #![allow(unsafe_code)]
+#![warn(clippy::std_instead_of_alloc)]
```

Comment:
Oooh, can we add these to other no_std crates?

Reply:
if we do, sounds like we need a new house rule?

Reply:
I found these useful mostly to do the conversion (you can add these, add `external crate alloc`, and do `cargo clippy --fix`, and it gets you most of the way there). Beyond this, they're mostly useful to ensure that we don't require the `std` feature in places where it's not really necessary.